proto/firecracker.proto

```diff
@@ -135,3 +137,44 @@ message JailerConfig {
     DriveExposePolicy DriveExposePolicy = 7;
 }
 
+message UpdateBalloonRequest {
+    string VMID = 1;
+    int64 AmountMb = 2;
+}
+
+message GetBalloonConfigRequest{
+    string VMID = 1;
+}
+
+message GetBalloonConfigResponse{
+    FirecrackerBalloonDevice BalloonConfig = 6;
```

Comment:
The number on the right hand side is the order of the fields, to be backward-compatible while adding/removing fields. So you should start from 1 here.

Reply:
Good catch, thanks. Fixed it.